### ISSUE TITLE
"inconsistent use of library definitions" writes no line

### ISSUE THREAD

**Issue by runn1ng** [facebook/flow#801]
I got this error with flow

    /home/karelb/dev/chrome-extension/src/tasks/receive.js: inconsistent use of library definitions
    /tmp/flow/flowlib_2cfdfc8a/core.js:410:1,414:1: ArrayBuffer
    This type is incompatible with
    /tmp/flow/flowlib_2cfdfc8a/core.js:437:1,446:1: Uint8Array

However, it's hard to find exactly where the inconsistent use was, since mostly any edit will cause dozens of other errors elsewere. Other type errors usually write exact place, but this does not.

**Comment by samwgoldman** [comment]
Thanks for the report. Can you also provide a reduced code sample that reproduces this error message? Improving error messages is definitely on our radar, and it will help to have examples we can check against to ensure our output is helpful.

For now, it might help you do know that you can use the `--traces` argument to Flow to get more error output. The additional output is a bit hard to parse, but it should point to the relevant line in your source code.

**Comment by runn1ng** [comment]
It's actually an open source project so I can

The problem was at this file

https://github.com/trezor/trezor-chrome-extension/blob/390df4dd85cfc3e6ca7dfbf3054d69c4d57665ba/src/tasks/receive.js

Lines 122-123
ParseFirstInput expects Uint8Array (as defined on line 72; maybe leftover
from some older version), but the hid.receive returns Promise<ArrayBuffer>,
as defined here

https://github.com/trezor/trezor-chrome-extension/blob/390df4dd85cfc3e6ca7dfbf3054d69c4d57665ba/src/chrome/hid.js
Line 92

It was fixed by replacing the Uint8Array in the function definition to
ArrayBuffer.

**Comment by samwgoldman** [comment]
I'm going to go out on a limb here and say this issue is resolved. I admit that I didn't actually try the linked code samples (out of laziness) but I just closed a dozen or so related issues that are all tested resolved. Please reply if you think I've closed this in error.
